spot_wrapper/spot_arm.py

```diff
@@ -0,0 +1,609 @@
+                self.wait_for_arm_command_to_complete(cmd_id)
+        except Exception as e:
+            return False, f"Exception occured during arm movement: {e}"
+
+        return True, "Moved arm successfully"
+
+    def gripper_open(self) -> typing.Tuple[bool, str]:
+        """
+        Fully opens the gripper
+
+        Returns:
+            Boolean success, string message
+        """
+        try:
+            success, msg = self.ensure_arm_power_and_stand()
+            if not success:
+                self._logger.info(msg)
+                return False, msg
+            else:
+                # Open gripper
+                command = RobotCommandBuilder.claw_gripper_open_command()
+
+                # Command issue with RobotCommandClient
+                cmd_id = self._robot_command_client.robot_command(command)
+                self._logger.info("Command gripper open sent")
+                self.wait_for_arm_command_to_complete(cmd_id)
+
+        except Exception as e:
+            return False, f"Exception occured while gripper was moving: {e}"
+
+        return True, "Open gripper success"
+
+    def gripper_close(self) -> typing.Tuple[bool, str]:
+        """
+        Closes the gripper
+
+        Returns:
+            Boolean success, string message
+        """
+        try:
+            success, msg = self.ensure_arm_power_and_stand()
+            if not success:
+                self._logger.info(msg)
+                return False, msg
+            else:
+                # Close gripper
+                command = RobotCommandBuilder.claw_gripper_close_command()
+
+                # Command issue with RobotCommandClient
+                cmd_id = self._robot_command_client.robot_command(command)
+                self._logger.info("Command gripper close sent")
+                self.wait_for_arm_command_to_complete(cmd_id)
+
+        except Exception as e:
+            return False, f"Exception occured while gripper was moving: {e}"
+
+        return True, "Closed gripper successfully"
+
+    def gripper_angle_open(self, gripper_ang: float) -> typing.Tuple[bool, str]:
+        """
+        Takes an angle between 0 (closed) and 90 (fully opened) and opens the gripper at this angle
+
+        Args:
+            gripper_ang: Angle to which the gripper should be opened
+
+        Returns:
+            Boolean success, string message
+        """
+        if gripper_ang > 90 or gripper_ang < 0:
+            return False, "Gripper angle must be between 0 and 90"
+        try:
+            success, msg = self.ensure_arm_power_and_stand()
+            if not success:
+                self._logger.info(msg)
+                return False, msg
+            else:
+                # The open angle command does not take degrees but the limits
+                # defined in the urdf, that is why we have to interpolate
+                closed = 0.349066
+                opened = -1.396263
```

Comment:
This is interesting because the [robot command builder](https://github.com/boston-dynamics/spot-sdk/blob/master/python/bosdyn-client/src/bosdyn/client/robot_command.py#L32) uses:
`_CLAW_GRIPPER_OPEN_ANGLE = -1.5708
_CLAW_GRIPPER_CLOSED_ANGLE = 0
`

Reply:
These changes look to have been made by @maubrunn, not sure why they are different.